README.md

```diff
@@ -0,0 +1,127 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Welcome...</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [PORTFOLIO] <a name="about-project"></a>
+
+My PORTFOLIO project in Microverse.
+
+**[PORTFLIO]** is a project experience that Microverse offers to help nuture your skills in designing on small screens.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+
+
+### Key Features <a name="key-features"></a>
+
+- **[prints out a web page of a portfolio]**
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+An extension to your browser
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone [https://github.com/2004-okari/MY-PORTFOLIO](https://github.com/2004-okari/MY-PORTFOLIO)
+### Install
+
+Install this project with:
+
+cd MY-PORTFOLIO
+gem install
```

Comment:
- Hey, this is a html-css project, you should use the `npm install` command which will install all the dependencies used and it's not a ruby or rails project. please, update this command.